portfolio/src/main/webapp/index.html

```diff
@@ -1,59 +1,88 @@
     <script src="script.js"></script>
   </head>
 
-  <body style="background-image: url('/images/background.jpg');">
+  <body onload=initBody() style="background-image: url('/images/background.jpg');">
       <div class=”row”>
         <h1 class="icons">
-            <a href="https://github.com/BenLin00/">
+            <a href="https://github.com/BenLin00/" target="_blank">
                 <img src="https://img.icons8.com/clouds/100/000000/github.png"> </a>
-            <a href="https://www.instagram.com/benlinja/?hl=en">
+            <a href="https://www.instagram.com/benlinja/?hl=en" target="_blank">
                 <img src="https://img.icons8.com/clouds/100/000000/instagram-new.png"/> </a>
-            <a href="https://open.spotify.com/user/2255uj7eayjdhgwxnk2qn6swq?si=YURi3l3kSJm8pXESI9SJbA">
+            <a href="https://open.spotify.com/user/2255uj7eayjdhgwxnk2qn6swq?si=YURi3l3kSJm8pXESI9SJbA" target="_blank">
                 <img src="https://img.icons8.com/clouds/100/000000/spotify.png"/> </a>
+            <a href="mailto:blin1@umd.edu" target="_blank">
+                <img src="https://img.icons8.com/clouds/100/000000/gmail.png"/> </a>
+            <a href="/images/Lin_Benjamin_Resume.pdf" target="_blank">
+                <img src="https://img.icons8.com/clouds/100/000000/document.png"/> </a>
         </h1>
         
-
-        <div class="col-sm">
+<!-- left column containing linkedin profile card -->
+        <div class="column left">
         Connect with me on LinkedIn!
         <div class="LI-profile-badge"  data-version="v1" data-size="medium" data-locale="en_US" data-type="vertical" data-theme="light" data-vanity="benlin-umd"><a class="LI-simple-link" href='https://www.linkedin.com/in/benlin-umd?trk=profile-badge'>Benjamin (Ben) Lin</a></div>
-        </div>
         <br>
         Some of my good vibe music!~
         <br>
         <iframe src="https://open.spotify.com/embed/playlist/4Im4I4bGUIdihq6cL872mV" width="300" height="380" frameborder="0" allowtransparency="true" allow="encrypted-media"></iframe>
-
-        <div class="col-sm">
-        <p>Click here to get a random message:</p>
-        <button onclick="addRandomGreeting()">Hello</button>
-        <div id="greeting-container"></div>
-        </div>
       
-        <div id="content" class="col-sm">
-        <h1> My name is Ben Lin </h1>
-        <br>
-        I'm a rising 3rd year student at University of Maryland studying Computer Science
-        <br>
-        Here's a picture of me!
-        <br>
-        <img src="/images/headshot.png">
         </div>
     
-        <p> <img src="/images/meme.jpg" > </p>
+<!-- center column with my photo -->
+        <div class="column middle">
+            <h1> My name is Ben Lin </h1>
+            <br>
+            I'm a rising 3rd year student at University of Maryland studying Computer Science
+            <br>
+            Here's a picture of me!
+            <br>
+                <img src="/images/me.jpg" width="300">
```

Comment:
I would try to keep spacing more consistent here

Reply:
Thanks for the catch!